frontend/src/views/Home.vue

```diff
@@ -1,7 +1,63 @@
 <template>
   <div class="home">
-    <p>TEST</p>
-    <router-link :to="{ name : 'Jeu' }">Jouer</router-link>
+    <h1>Yahtzee</h1>
+    <router-link :to="{ name : 'Jeu' }">
+      <div class = "rectangle">
+        2 joueurs
+      </div>
+    </router-link>
```

Comment:
Mettre le `router-link` hors du `div` fait que toute la largeur est clickable. Cela semble se régler en faisant ce changement: 

```suggestion
    <router-link class="rectangle" :to="{ name : 'Jeu' }">
            2 joueurs
    </router-link>
```

Il n'y alors que le rectangle qui est clickable.